packages/block-editor/src/components/rich-text/style.scss

```diff
@@ -55,6 +55,7 @@ figcaption.block-editor-rich-text__editable [data-rich-text-placeholder]::before
 
 	.components-popover__content {
 		min-width: auto;
+		margin-bottom: 6px;
```

Comment:
while this seem simpler, from a user's perspective, it's less obvious. I wonder if keeping the prop and make it work by adding a margin could be the best of both worlds. No urgency though, just a thought in case this becomes a common pattern.

Reply:
From my experience with #18779, it does seem easier at first to have an option like this, but this option is way less flexible than CSS. In CSS you can adjust it based on classes etc. It's also something that's clearly styling and should ideally be done in CSS if possible.